<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalpaMainWindow</class>
 <widget class="QMainWindow" name="TalpaMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>722</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Talpa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadModel"/>
    <addaction name="actionSaveModel"/>
    <addaction name="separator"/>
    <addaction name="actionExportKiteScene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionChangeExtent"/>
    <addaction name="actionChangeLos"/>
    <addaction name="actionLoadReferenceScene"/>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
    <addaction name="separator"/>
    <addaction name="actionConfiguration"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Talpa"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionMisfitScene"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Online Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Logger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout_Talpa">
   <property name="text">
    <string>About Talpa</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="text">
    <string>Save model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="text">
    <string>Load model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExportKiteScene">
   <property name="text">
    <string>Export kite.Scene</string>
   </property>
  </action>
  <action name="actionLoadReferenceScene">
   <property name="text">
    <string>Load Reference Scene</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChangeExtent">
   <property name="text">
    <string>Change SandboxScene extent</string>
   </property>
  </action>
  <action name="actionChangeLos">
   <property name="text">
    <string>Change SandboxScene LOS</string>
   </property>
  </action>
  <action name="actionMisfitScene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Model residual</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TalpaMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
